tests/integration/components/nypr-account-forms/login-test.js

```diff
@@ -14,7 +14,7 @@ test('it renders', function(assert) {
 });
 
 test('submitting the form passes the login values to the authenticator', function(assert) {
-  let authenticate = sinon.stub().returns(RSVP.Promise.reject({}));
+  let authenticate = sinon.stub().returns(RSVP.Promise.resolve({}));
```

Comment:
This mock returned a rejected Promise when I originally wrote this, because at one point in the life of this component a successful form submission triggered a redirect. I wanted to avoid redirects in an integration test. 

That redirect situation changed but I never looked back at this test until now because it still passed.

It seems like newer versions of ember testing are not happy with the rejection, so now returning a resolved Promise.